### ISSUE TITLE
[Bug] Cannot use S3 as a remote manifest path

### ISSUE THREAD

**Issue by ogescalante** [astronomer/astronomer-cosmos#1126]
### Astronomer Cosmos Version

Other Astronomer Cosmos version (please specify below)

### If "Other Astronomer Cosmos version" selected, which one?

1.6.0a3

### dbt-core version

1.8.1

### LoadMode

CUSTOM

### ExecutionMode

KUBERNETES

### InvocationMode

None

### airflow version

2.9.2

### Operating System

Debian GNU/Linux 12 (bookworm)

### Deployment

Official Apache Airflow Helm Chart

### What happened?

I cannot see how to use S3 as remote location for my Manifest file, I keep getting this error:
<img width="749" alt="image" src="https://github.com/user-attachments/assets/ce3620a8-e598-497f-a73d-f14eca1da94d">
My dag code:
```py
from datetime import datetime
from airflow.models import Variable
from airflow import DAG
from cosmos import (
    ProjectConfig,
    ExecutionConfig,
    DbtTaskGroup,
    ExecutionMode,
    RenderConfig,
    LoadMode,
    ProfileConfig
)

import include.constants as ct

with DAG(
    dag_id="artemis_dag",
    start_date=datetime(2023, 7, 10),
    schedule_interval=None,
    default_args=ct.DEFAULT_ARGS,
    catchup=False,
    tags=["Artemis Pipeline"],
    description="Orchestrating the DBT models of the artemis-pipeline project",
    max_active_runs="{{ var.value.get('artemis_active_runs', 1 }}",
    max_active_tasks="{{ var.value.get('artemis_max_active_tasks', 32 }}",
) as dag:
    run_artemis_pipeline = DbtTaskGroup(
        group_id="artemis_pipeline",
        render_config=RenderConfig(load_method=LoadMode.DBT_MANIFEST),
        project_config=ProjectConfig(
            dbt_project_path=ct.ARTEMIS_PROJECT_DIR,
            manifest_path="s3://***/manifest.json"
        ),
        execution_config=ExecutionConfig(
            execution_mode=ExecutionMode.KUBERNETES,
        ),
        operator_args={
            "image": Variable.get("DBT_IMAGE"),
            "namespace": Variable.get("NAMESPACE"),
            "get_logs": True,
            "is_delete_operator_pod": True,
        },
    )

    run_artemis_pipeline
```

I've installed the astronomer-cosmos[amazon] but the problem persists.
If I try to use the `manifest_conn_id` the ui says this argument does not exists.

### How to reproduce

try creating a DAG passing s3 as the manifest path location.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

**Comment by pankajkoti** [comment]
hi @ogescalante , we have not included the PR https://github.com/astronomer/astronomer-cosmos/pull/1109 that adds support for using remote manifest paths yet in any of our pre-releases. With `1.6.0a3`, it is still trying to consider your remote path as a local path and is also not accepting the parameter `manifest_conn_id` as it was only introduced in the PR #1109 which is yet to be released/pre-released. I will discuss with the team if we can create a pre-release sometime soon that includes the PR so you can then test it.

**Comment by ogescalante** [comment]
Hi @pankajkoti  thanks for getting back on this, consider please removing it from the official docs meanwhile since it is pretty frustrating to not being able to do this.
https://astronomer.github.io/astronomer-cosmos/configuration/parsing-methods.html

**Comment by ogescalante** [comment]
For those with the same problem, my workarounf was implementing an s3 hook myself:
```py
from datetime import datetime
from airflow.models import Variable
from airflow import DAG
from cosmos import (
    ProjectConfig,
    ExecutionConfig,
    DbtTaskGroup,
    ExecutionMode,
    RenderConfig,
    LoadMode,
    ProfileConfig
)
from airflow.hooks.S3_hook import S3Hook


import include.constants as ct

manifest_path = ct.ARTEMIS_PROJECT_DIR / "target" / "manifest.json"

def download_file_from_s3():
    s3 = S3Hook()
    bucket_name = ***
    file_key = 'manifest.json'

    s3.get_key(file_key, bucket_name).download_file(manifest_path)

with DAG(
    dag_id="artemis_dag",
    start_date=datetime(2023, 7, 10),
    schedule_interval=None,
    default_args=ct.DEFAULT_ARGS,
    catchup=False,
    tags=["Artemis Pipeline"],
    description="Orchestrating the DBT models of the artemis-pipeline project",
    max_active_runs="{{ var.value.get('artemis_active_runs', 1 }}",
    max_active_tasks="{{ var.value.get('artemis_max_active_tasks', 32 }}",
) as dag:
    run_artemis_pipeline = DbtTaskGroup(
        group_id="artemis_pipeline",
        render_config=RenderConfig(load_method=LoadMode.DBT_MANIFEST),
        project_config=ProjectConfig(
            dbt_project_path=ct.ARTEMIS_PROJECT_DIR,
            manifest_path=manifest_path
        ),
        execution_config=ExecutionConfig(
            execution_mode=ExecutionMode.KUBERNETES,
        ),
        operator_args={
            "image": Variable.get("DBT_IMAGE"),
            "namespace": Variable.get("NAMESPACE"),
            "get_logs": True,
            "is_delete_operator_pod": True,
        },
    )

    run_artemis_pipeline
```

**Comment by pankajastro** [comment]
Yes, the docs is confusing for the user at the moment. I have created an issue to fix it https://github.com/astronomer/astronomer-cosmos/issues/1128

**Comment by pankajkoti** [comment]
@ogescalante Thanks for voicing out. Apologies for the frustration caused due to the docs rendering from main branch. I just took a relook at the docs, and they do mention that this will only be available since and after Cosmos 1.6 which is yet to be released. 

Thanks @pankajastro for creating an issue to fix the docs.

**Comment by pankajkoti** [comment]
hi @ogescalante we just have created a pre-release 1.6.0a4 https://pypi.org/project/astronomer-cosmos/1.6.0a4/ that includes the relevant PR for remote manifest load. Appreciate if you would like to test it and provide any feedback! :)

**Comment by pankajkoti** [comment]
The ability to use S3 for remote manifest has been released with Cosmos 1.6.0 release. I invite you test it out. I'm closing this at the moment, but feel free to re-open in case you observe any issue.
